File: app/src/main/java/com/BlizzardArmory/model/warcraft/charactersummary/MythicKeystoneProfile.java
package com.BlizzardArmory.model.warcraft.charactersummary;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * The type Mythic keystone profile.
 */
public class MythicKeystoneProfile {

    @SerializedName("href")
    @Expose
    private String href;

    /**
     * Gets href.
     *
     * @return the href
     */
    public String getHref() {
        return href;
    }

    /**
     * Sets href.
     *
     * @param href the href
     */
    public void setHref(String href) {
        this.href = href;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/charactersummary/MythicKeystoneProfile.kt
package com.BlizzardArmory.model.warcraft.charactersummary

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Mythic keystone profile.
 */
@Keep
data class MythicKeystoneProfile(

    @SerializedName("href")
    var href: String

)